src/functions/template-tags/event.php

```diff
@@ -100,27 +100,29 @@ function tribe_get_event( $event = null, $output = OBJECT, $filter = 'raw', $for
 
 		$post = false;
 
-		if ( ! $force ) {
-			$cache_post    = get_post( $event );
-			if ( empty( $cache_post ) ) {
-				return null;
-			}
+		/** @var Tribe__Cache $cache */
+		$cache = tribe( 'cache' );
+
+		$cache_post    = get_post( $event );
+		if ( empty( $cache_post ) ) {
+			return null;
+		}
+
+		$key_fields    = [
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: c7e8917c3431a18ef0a2eebafbf45da1</sub>